src/types/job.rs

```diff
@@ -33,7 +33,10 @@ pub struct JobDescriptor {
     pub pass: bool,
     pub msg: String,
     pub date: String,
+    // don't deprecate until `ecosystems` is live.
+    //#[deprecated = "Use `ecosystems` to support multiple ecosystems."]
     pub ecosystem: String,
+    pub ecosystems: Vec<String>,
```

Comment:
Similar to `ecosystem`, this should be optional for deserialization. That way we can immediately merge this change into CLI without breaking compatibility with current API in prod (which obviously isn't sending the `ecosystems` field yet).

```suggestion
    #[serde(default)]
    pub ecosystems: Vec<String>,
```

Reply:
Good catch. I'd thought about making this default to a vector containing the content of `ecosystem`, but it doesn't seem like there's a reasonable way to do that when using serde_derive.